packages/element-lib/package.json

```diff
@@ -40,13 +40,17 @@
     "@transmute/did-wallet": "0.0.0-6",
     "@transmute/es256k-jws-ts": "0.0.4",
     "@truffle/contract": "^4.2.14",
-    "@truffle/hdwallet-provider": "1.0.38-next.0",
+    "@truffle/hdwallet-provider": "^1.0.40",
+    "@types/webassembly-js-api": "0.0.2",
+    "acorn": "^7.3.1",
```

Comment:
What does this dep do?

Reply:
were required to install manually after running npm audit fix ... the subassembly module was due to an element module I think it was EC256K.